GitUI/CommandsDialogs/BrowseDialog/DashboardControl/UserRepositoriesListController.cs

```diff
@@ -8,12 +8,49 @@
 
 namespace GitUI.CommandsDialogs.BrowseDialog.DashboardControl
 {
+    public interface IRepositoryFilter
+    {
+        void Filter(IList<Repository> repositories);
+    }
+
+    /// <summary>
+    /// Case insensitive text search based on the repository path
+    /// </summary>
+    public class RepositoryTextFilter : IRepositoryFilter
+    {
+        private readonly string _pattern;
+
+        public RepositoryTextFilter(string pattern)
+        {
+            _pattern = pattern;
```

Comment:
Sorry but looking at it again, it seems weird to me...
As long as you trim the string to check the length, the `.Trim()` must be here, no!?!

Either we don't trim or we use the trimmed string as the filter pattern, right?

Reply:
Yes, you're right. It probably would make sense to trim the spaces only from the left (to ignore accidentially added spaces at the beginning of the input) but keep spaces on the right since it's legitim to have a repo path with spaces in it